### ISSUE TITLE
Cannot execute WoK with gingers390x plug-in

### ISSUE THREAD

**Issue by danielhb** [kimchi-project/gingers390x#1]
This is the output of a simple WoK run with the s390x in the source tree:

```
[danielhb@arthas kimchi]$ sudo ./src/wokd --environment=dev
Traceback (most recent call last):
  File "./src/wokd", line 101, in <module>
    sys.exit(main(sys.argv[1:]))
  File "./src/wokd", line 98, in main
    wok.server.main(options)
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/server.py", line 208, in main
    srv = Server(options)
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/server.py", line 137, in __init__
    self._load_plugins(options)
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/server.py", line 160, in _load_plugins
    plugin_app = import_class(plugin_class)(options)
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/plugins/gingers390x/gingers390x.py", line 38, in __init__
    self.model = gingerS390xModel.Model()
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/plugins/gingers390x/model/model.py", line 46, in __init__
    self.objstore = ObjectStore(objstore_loc)
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/objectstore.py", line 91, in __init__
    self._init_db()
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/objectstore.py", line 94, in _init_db
    conn = self._get_conn()
  File "/home/danielhb/kimchi/wok/kimchi/src/wok/objectstore.py", line 117, in _get_conn
    timeout=10)
sqlite3.OperationalError: unable to open database file
[danielhb@arthas kimchi]$ 
```

This is a similar issue which @chandrureddy  fixed in Kimchi/WoK a few days ago. I believe the solution is similar.

**Comment by sureshab** [comment]
Hi @danielhb! 
I have fixed this issue, will be sending the patch shortly.

**Comment by danielhb** [comment]
bug fixed by 4d9b9b3.
